### ISSUE TITLE
There is a small pause added in between tracks

### ISSUE THREAD

**Issue by mutageneral** [xou816/spot#502]
**Describe the bug**
There is a small pause added in between tracks. It is hearable on albums that have songs that flow from one song to the other with no  silence. These pauses are not present in the official app.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any album that has songs that flows from one to the next with no pause. A good example is the [High Hell Soundtrack](https://open.spotify.com/album/6lbjCIfLLNDXWuc38aJcaE).
2. Start the first song in the [High Hell Soundtrack](https://open.spotify.com/album/6lbjCIfLLNDXWuc38aJcaE) album and skip toward the last 10 seconds of the song.
3. Wait until the song end and the next one starts, and you will notice an unnecessary pause. If you do the same in your web browser or the official app, you will notice the next song was supposed to immediately start playing.

**Expected behaviour**
For the annoying pauses to not be present and for the next track to start playing immediately like on the official app so that I can enjoy my music the way the artist intended.

**General information:**
 - Distribution: Fedora 35
 - Installation method: Flathub
 - Version: 0.3.1
 - Device used: Laptop

**Stack trace:**
The only thing here was the same unrelated GTK warning that showed up over and over again.
```
(spot:2): Gtk-WARNING **: 16:19:13.332: GtkGizmo 0x55e775aadc00 (highlight) reported min width -2, but sizes must be >= 0
```

**Comment by ondras12345** [comment]
Duplicate of https://github.com/xou816/spot/issues/193

**Comment by xou816** [comment]
fixed by #532  possibly :) feel free to reopen if needed
